opacus/tests/multigpu_gradcheck.py

```diff
@@ -77,7 +80,7 @@ def demo_basic(rank, weight, world_size, dp, clipping):
     if dp:
         max_grad_norm = 1e8
```

Comment:
Let's follow [pep-8](https://www.python.org/dev/peps/pep-0008/#constants) and name this constant with capital letters: `MAX_GRAD_NORM`. This will emphasize that this thing is pre-defined and hard coded.

Reply:
It's not a constant: if clipping is per-layer, it becomes a list, see line 80.